hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/SQLFederationStateStore.java

```diff
@@ -258,30 +256,26 @@ public SubClusterRegisterResponse registerSubCluster(
 
       // Check the ROWCOUNT value, if it is equal to 0 it means the call
       // did not add a new subcluster into FederationStateStore
-      if (cstmt.getInt(9) == 0) {
-        String errMsg = "SubCluster " + subClusterId
-            + " was not registered into the StateStore";
+      int rowCount = cstmt.getInt("rowCount_OUT");
+      if (rowCount == 0) {
+        String errMsg = "SubCluster " + subClusterId + " was not registered into the StateStore";
         FederationStateStoreUtils.logAndThrowStoreException(LOG, errMsg);
```

Comment:
Didn't we have the one with the format?

Reply:
Thank you for your help in reviewing the code, I will refactor this part of the code and use the format method.